src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -196,9 +196,9 @@ public function setRequired(bool $boolean)
     /**
      * Sets this node as deprecated.
      *
-     * @param string $package The name of the composer package that is triggering the deprecation
-     * @param string $version The version of the package that introduced the deprecation
-     * @param string $message the deprecation message to use
+     * @param ?string $package The name of the composer package that is triggering the deprecation
```

Comment:
Other phpdoc use the union type with null.
```suggestion
     * @param string|null $package The name of the composer package that is triggering the deprecation
```